Subject: nqptp: bump to version 1.2.9

1.2.9 serves several Shairport Sync instances on one host (nqptp#50, a shared
memory region per client); 1.2.4 serves one. openHC runs one AirPlay 2 receiver
per audio endpoint, all on the one nqptp. Drop this once Buildroot carries
1.2.9 or later.

diff -ruN a/package/nqptp/nqptp.hash b/package/nqptp/nqptp.hash
--- a/package/nqptp/nqptp.hash
+++ b/package/nqptp/nqptp.hash
@@ -1,3 +1,3 @@
 # Locally calculated
-sha256  1df1d5edd5b713010d6495b3abca4c1cf4ad8fa6029df0abeb9e4de8e0eb707a  nqptp-1.2.4.tar.gz
+sha256  6aa1d73b8ee760bf19378c319db0d0537a867ff961764b150e19f87d96a02210  nqptp-1.2.9.tar.gz
 sha256  8177f97513213526df2cf6184d8ff986c675afb514d4e68a404010521b880643  LICENSE
diff -ruN a/package/nqptp/nqptp.mk b/package/nqptp/nqptp.mk
--- a/package/nqptp/nqptp.mk
+++ b/package/nqptp/nqptp.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-NQPTP_VERSION = 1.2.4
+NQPTP_VERSION = 1.2.9
 NQPTP_SITE = $(call github,mikebrady,nqptp,$(NQPTP_VERSION))
 NQPTP_LICENSE = GPL-2.0
 NQPTP_LICENSE_FILES = LICENSE
